### PR TITLE
Comparison to flash gamma

### PR DESCRIPTION
**Pull request by SimonBiggs**
See https://github.com/samuelpeet/flashgamma/issues/7

### CODE REVIEW

File: examples/drafts/gamma/compare_to_flash_gamma.ipynb [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot_passing_rates(pass_rates)"
+   ]
+  },
+  {
```
Comment (sjswerdloff) [inline examples/drafts/gamma/compare_to_flash_gamma.ipynb:277]: might be nice to have additional markdown here to separate the pymedphys calculation from the plotting of the FlashGamma pass rates

File: setup.py [hunk 2]
```diff
@@ -185,6 +185,7 @@
     "dicom": ["pydicom>=2.0.0", "pynetdicom", "pylibjpeg-libjpeg"],
     "docs": ["jupyter-book>=0.8.3", "sphinx-argparse", "sphinxcontrib-napoleon"],
     "doctests": ["tensorflow>=2.2.0", "black>=19.3b0,<20.0", "tomlkit"],
+    "external": ["flashgamma"],
```
Comment (sjswerdloff) [inline setup.py:188]: external is a very broad term.
in the property  (land/housing) world, this kind of thing is referred to as "comparables" or "comps".

**Review by sjswerdloff — APPROVED** [review]
assuming botocore issue can be resolved...
Might be a good time to update that to 1.19.27 (latest), unless there is a conflict

File: setup.py (hunk 2 above)
Comment (SimonBiggs) [reply setup.py:188]: > external is a very broad term.

Yeah, good call.

> in the property (land/housing) world, this kind of thing is referred to as "comparables" or "comps".

I like it :) Will make the change.

**Comment by SimonBiggs** [comment]
@sjswerdloff do you know what happened here? If you fixed it, thanks :)

**Comment by sjswerdloff** [comment]
Re-ran jobs.
It smelled like a timeout

File: examples/drafts/gamma/compare_to_flash_gamma.ipynb (hunk 1 above)
Comment (SimonBiggs) [reply examples/drafts/gamma/compare_to_flash_gamma.ipynb:303]: Thanks @sjswerdloff, I cleaned this up a bit :)

**Pull request merged**